scripts/ci/libraries/_remove_images.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Removes airflow CI and base images
+function remove_all_images() {
```

Comment:
What do you tink of having this, _push.sh and _pull.sh all in a single _images.sh file?

Reply:
Yeah. Push and remove were small. I combined them all into _push_pull_remove_images.sh. Since we also have _build_images.sh (and these have quite a bit more complex logic) I think it makes sense to split them this way.

I also considered splitting it across prod/ci images but there are too many common parts in build to split it further.